Sleep fixes + some new toy checks in setup

lotar232 opened the pull request:
Sleep fixes + some new toy checks in setup.... I tried GitHub desktop, but publishing from there doesn't seem to work so I uploaded it to a fork online and created a pull request from there (not sure why the diff of setup seems to show the whole file ?.???)

lotar232 committed:
Add files via upload

lotar232 committed:
Add files via upload

fixed some millisecond vs second issues in chat

lotar232 committed:
updates startup routine

lotar232 committed:
Add files via upload

implemented house rules and reporting exercise

GodDragoner commented on Chat/ChatUtil.js:
```diff
@@ -1,31 +1,31 @@
+    }
+
+    if(wait === undefined || wait) {
+        sleep(.2 + message.length * .01);
```
Please restore the previous state. It works like that (with the TimeUnit given as parameter)

lotar232 replied:
agree eventually... for debugging though having text go by very quickly is useful for running through long sessions to get to a buggy spot or testing all the paths through  the scripts.... can we leave it like this until its in a near usable state?

samm112 committed:
Fix startup sound directory

samm112 committed:
Add ass vocabulary and return slave string

samm112 committed:
Add bell sound util

samm112 committed:
Add Position Trainig variables

samm112 committed:
Fix Laugh sound directory

lotar232 committed:
Add files via upload

"rules" and "Exercise" pretty functional... on to dungeon next

lotar232 committed:
Add files via upload

updated lobby to access new areas

lotar232 committed:
Add files via upload

a few more vocabularies

lotar232 commented:
adds "rules" and "exercise" submodule... working on Dungeon next...

need to put Audio and images directory up somewhere with instructions on installing it... will do that soon.... should we start a spicy TAJ thread on milovana?

samm112 committed:
Implement Gold Utils

samm112 committed:
Add vocab for balls grin and happy emote

samm112 committed:
Implement simple position training and tests

samm112 committed:
Fix typo in personality.js

samm112 committed:
Add vocab js emotesad ready and slut

samm112 committed:
Complete complex position training

samm112 committed:
Remove duplication of training end

lotar232 committed:
Add files via upload

a slightly less elegant version of assistant lobby that seems to better handle call-return to the various modules without hanging...

lotar232 committed:
Add files via upload

Snapshotting local version of dungeon... still mostly non-functional... "soft" punishments are getting close to being functional

lotar232 committed:
Add files via upload

adding a fell-off response for the punishment that uses clips/clamps

lotar232 committed:
Add files via upload

minor updates

lotar232 committed:
Add files via upload

updating/adding some vocabularies...

lotar232 committed:
Merge pull request #1 from samm112/position-training

Position training

lotar232 committed:
Add files via upload

made some dungeon updates

lotar232 committed:
Add files via upload

punishment updates (still work in progress but most are working)

lotar232 committed:
Add files via upload

dungeon updates

lotar232 committed:
Add files via upload

adding a few more ported vocabularies used in punishments

GodDragoner committed:
Update ChatUtil.js

GodDragoner committed:
Update StartSession.js
